app/static/src/js/headscripts/gtm.js

```diff
@@ -23,6 +23,52 @@ function add_GTM() {
     GTM_Loaded = true;
 }
 
+function diagnosed(){
+    if (!window.sessionData) return; // If no session data available exit
+    window.dataLayer = window.dataLayer || [];
+
+    const path = window.location.pathname;
+
+    if (path.endsWith('/legal-aid-available') || path.includes('fast-tracked')) {
+        window.dataLayer.push({
+            event: 'diagnosed',
```

Comment:
Maybe extract slightly reorganise to save on duplication? feed in diagnosis_result?